### DISCUSSION TITLE
Proposal for new activities

### DISCUSSION THREAD

**Discussion by anchillabc** [littlesera/LSCG#262] [Ideas]
How about an activity to spin someone who is in suspension bondage?
Rope suspension, ceiling rope / chains would each be candidates for this.
Ideas for possible activities:

- Spin slowly
- Spin fast
- Nudge 
- Swing back and forth

Also, at least the swing back and forth activity might make sense at the arm area for those who are currently suspended.

**Comment by jimmyG1** [comment]
if thay are spun enough thay get dizzy and cant stand
